mindsdb/integrations/handlers/llama_index_handler/README.md

```diff
@@ -0,0 +1,27 @@
+## LlamaIndex Handler
+LlamaIndex ML handler for MindsDB, create indexes over data plugged in mindsdb and use it to create a Question & Answer (Q&A) systems
+
+## LlamaIndex
+LlamaIndex is a data framework for your LLM application.In this handler,python client of LlamaIndex api is used and more information about this python client can be found (here)[https://gpt-index.readthedocs.io/en/latest/]
+
+## Implemented Features
+- [x] LlamaIndex ML Handler
+  - [x] [Support Web Page Reader](https://gpt-index.readthedocs.io/en/latest/examples/data_connectors/WebPageDemo.html)
+
+## Example Usage
+~~~sql
+CREATE MODEL my_qa_model
+PREDICT answer
+USING 
+  engine = 'llama_index', 
+  index_class = 'GPTVectorStoreIndex',
+  query_engine = 'as_query_engine', 
```

Comment:
nit: is this configuration hooked up properly? In the code below I don't see this arg being used at all.

Reply:
Yes, I have used some of the args mentioned in the configuration. I have not used index_class & query_engine in the code